algebird-core/src/main/scala/com/twitter/algebird/HyperLogLog.scala

```diff
@@ -662,3 +687,15 @@ case class SetSizeAggregator[A](hllBits: Int, maxSetSize: Int = 10)(implicit toB
   val leftSemigroup = new HyperLogLogMonoid(hllBits)
   val rightAggregator = Aggregator.uniqueCount[A].andThenPresent { _.toLong }
 }
+
+case class SetSizeHashAggregator[A](hllBits: Int, maxSetSize: Int = 10)(implicit hash: Hash128[A])
```

Comment:
The level of duplication here with `SetSizeAggregator` bothers me. Can we not just introduce a backwards-compatible `apply` that takes a fn and wraps it in a `Hash128`?